plugin-build/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ plugins {
     id("java-gradle-plugin")
     id("com.vanniktech.maven.publish") version BuildPluginsVersion.MAVEN_PUBLISH apply false
     id("org.jlleitschuh.gradle.ktlint") version BuildPluginsVersion.KTLINT
+    id("com.stepango.aar2jar") version BuildPluginsVersion.AAR_2_JAR
```

Comment:
Since gradle plugin is a pure java project, I had to use this plugin to be able to include .aar dependencies